eventmesh-common/build.gradle

```diff
@@ -29,7 +29,7 @@ dependencies {
     implementation "org.apache.logging.log4j:log4j-core"
```

Comment:
suggest to specify a version.

Reply:
Why specify a separate version here, instead of using dependencyManagement's unified version management?and did not specify the version before

Reply:
That's ok!